easybuild/easyconfigs/r/ROOT/ROOT-v6.10.08-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,44 @@
+name = 'ROOT'
+version = 'v6.10.08'
```

Comment:
@Helios07 We stopped using the `v` in the version for recent `ROOT` easyconfigs (see https://github.com/easybuilders/easybuild-easyconfigs/pull/5370/files and https://github.com/easybuilders/easybuild-easyconfigs/pull/5081/files